alu_u32/src/bitwise/stark.rs

```diff
@@ -0,0 +1,57 @@
+use super::columns::Bitwise32Cols;
+use super::Bitwise32Chip;
+use core::borrow::Borrow;
+use valida_machine::MEMORY_CELL_BYTES;
+
+use p3_air::{Air, AirBuilder};
+use p3_field::AbstractField;
+use p3_matrix::MatrixRows;
+
+impl<F, AB> Air<AB> for Bitwise32Chip
+where
+    F: AbstractField,
+    AB: AirBuilder<F = F>,
+{
+    fn eval(&self, builder: &mut AB) {
```

Comment:
Seems like we also need to range check bits_1 and bits_2 to ensure a correct decomposition. Could use `assert_bool` for it.

Reply:
Oops, I had those, but accidentally deleted when refactoring. Will add back.

Reply:
I think we should also `assert_bool` the opcode flags as well their sum, otherwise a prover might be able to make this table "receive" a different opcode by writing it as some combination of the bitwise opcodes?

Reply:
The sum should be `assert_one`, right?

Reply:
Just padding rows would have 0, no?

Reply:
Right. I also realized that `count` needs to be changed to the sum of the opcode flags to account for padding. Will fix both in the shift chip PR.